### ISSUE TITLE
flow direction

### ISSUE THREAD

**Issue by kfischerar** [QGEP/QGEP#214]
exists a tool to change the flow direction of a reach ?

if you visualze with arrows it looks pretty strange (and its wrong as well).
no change in the database, just "graphically switch" from/to

**Comment by m-kuhn** [comment]
Is there a reason not to change it in the database?

**Comment by urskaufmann** [comment]
There must be a change in the database. The graphics (point, line, ...)  come out of the database, there can not be a difference.

**Comment by andreasneumann** [comment]
I agree with Mathias and Urs. It needs to be changed in the database.

**Comment by Marpingen** [comment]
Where is the flow direction saved in the database?

**Comment by m-kuhn** [comment]
it's implicitly given from the order of points in the LineString.

**Comment by sjib** [comment]
Topology is defined with from_node_point and to_node_point in the reach.

**Comment by sjib** [comment]
Check also this topic https://github.com/QGEP/QGEP/issues/181
